crates/holochain_integrity_types/src/countersigning.rs

```diff
@@ -219,15 +257,37 @@ impl PreflightRequest {
         &mut self.signing_agents
     }
 
+    /// Optional signing agents accessor.
```

Comment:
Accessors aren't needed if the fields are pub which they should be in this crate